0847_deducing_this/deducing-this.bs

```diff
@@ -1993,4 +2049,9 @@ The authors would like to thank:
 - Chandler Carruth for a lot of feedback and guidance around design issues
 - Graham Heynes, Andrew Bennieston, Jeff Snyder for early feedback regarding the meaning of `this` inside function bodies
 - Amy Worthington, Jackie Chen, Vittorio Romeo, Tristan Brindle, Agustín Bergé, Louis Dionne, and Michael Park for early feedback
-- Guilherme Hartmann for his guidance with the implementation
\ No newline at end of file
+- Guilherme Hartmann for his guidance with the implementation
+- Richard Smith and Jens Weller for help with wording
```

Comment:
Jens Maurer. Also need to add Hubert Tong.

Reply:
done